### PR TITLE
Happy 2016!

### PR DESCRIPTION
**Pull request by drcrallen**

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Druid - a distributed column store.
-Copyright 2012-2015 Metamarkets Group Inc.
-Copyright 2015 Yahoo! Inc.
+Copyright 2012-2016 Metamarkets Group Inc.
+Copyright 2015-2016 Yahoo! Inc.
```
Comment (fjy) [inline NOTICE]: can you also add Imply?

Comment (drcrallen) [inline NOTICE]: I just updated dates in this one, I think any new (C) notices should be a separate PR

**Comment by fjy** [comment]
@drcrallen this is rebased from latest master?
I see Failed tests: 
  RemoteTaskRunnerTest.testRunExistingTaskThatHasntStartedRunning:157 expected:<SUCCESS> but was:<FAILED>
Tests run: 194, Failures: 1, Errors: 0, Skipped: 0

**Comment by drcrallen** [comment]
it's a 2 line diff that's a problem in master's test

**Comment by fjy** [comment]
@drcrallen yes just double checking you see this error after basing off of a versionof master that has @nishantmonu51's fix

**Comment by himanshug** [comment]
@fjy y@nishantmonu51 yes that test is failing in master , just now, i noticed that in a failure as well.

```
Tests run: 16, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 31.673 sec <<< FAILURE! - in io.druid.indexing.overlord.RemoteTaskRunnerTest

testRunExistingTaskThatHasntStartedRunning(io.druid.indexing.overlord.RemoteTaskRunnerTest)  Time elapsed: 1.733 sec  <<< FAILURE!

java.lang.AssertionError: expected:<SUCCESS> but was:<FAILED>

	at org.junit.Assert.fail(Assert.java:88)

	at org.junit.Assert.failNotEquals(Assert.java:743)

	at org.junit.Assert.assertEquals(Assert.java:118)

	at org.junit.Assert.assertEquals(Assert.java:144)

	at io.druid.indexing.overlord.RemoteTaskRunnerTest.testRunExistingTaskThatHasntStartedRunning(RemoteTaskRunnerTest.java:157)

```

**Comment by himanshug** [comment]
:+1:
